### PR TITLE
[MSGINA] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/msgina/lang/ro-RO.rc [hunk 1]
```diff
@@ -164,11 +164,11 @@ BEGIN
     LTEXT "", IDC_REASON_DESCRIPTION, 39, 184, 190, 27
     LTEXT "Co&mentariu:", IDC_STATIC, 39, 214, 80, 8
     EDITTEXT IDC_REASON_COMMENT, 39, 224, 190, 35, ES_MULTILINE | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "Con&firmă", IDOK, 150, 276, 55, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", IDCANCEL, 211, 276, 55, 14
+    DEFPUSHBUTTON "&OK", IDOK, 150, 276, 55, 14, WS_GROUP
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:167]: This introduces an accelerator collision with the line above
 LTEXT "&Opțiune:", IDC_STATIC, 39, 154, 80, 8

File: dll/win32/msgina/lang/ro-RO.rc [hunk 2]
```diff
@@ -58,12 +58,12 @@ BEGIN
...
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
+    PUSHBUTTON "&Deautentificare…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
+    PUSHBUTTON "Închid&erer…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
+    PUSHBUTTON "Schim&bare parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:64]: This line introduces an accelerator collision with the line
 PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

**Comment by AndreiMiloiu** [comment]
Thank you @JoachimHenze! I fixed the two issues.

**Comment by JoachimHenze** [comment]
> Thank you @JoachimHenze! I fixed the both issues.

No you didn't fix that properly! Now 
 PUSHBUTTON "Bloca&re calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
collides with
PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14

File: dll/win32/msgina/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: No you didn't fix that properly! Now 
 PUSHBUTTON "Bloca&re calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
collides with
PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14

File: dll/win32/msgina/lang/ro-RO.rc [hunk 3]
```diff
@@ -164,11 +164,11 @@ BEGIN
     LTEXT "", IDC_REASON_DESCRIPTION, 39, 184, 190, 27
     LTEXT "Co&mentariu:", IDC_STATIC, 39, 214, 80, 8
     EDITTEXT IDC_REASON_COMMENT, 39, 224, 190, 35, ES_MULTILINE | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "Con&firmă", IDOK, 150, 276, 55, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", IDCANCEL, 211, 276, 55, 14
+    DEFPUSHBUTTON "O&K", IDOK, 150, 276, 55, 14, WS_GROUP
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:167]: It's very unlikely that Windows sometimes uses the &OK and sometimes the O&K. I bet MS would try to avoid such a thing. I do dare to claim that you did not actually double-check the Romanian Windows here. I do request a screenshot of your Romanian Windows for this one before I will undo my rejection.

File: dll/win32/msgina/lang/ro-RO.rc [hunk 4]
```diff
@@ -43,9 +43,9 @@ BEGIN
     EDITTEXT IDC_LOGON_PASSWORD, 60, 76, 155, 14, ES_AUTOHSCROLL | ES_PASSWORD
     LTEXT "Vă autentificați la:", IDC_STATIC, 6, 96, 52, 8
     COMBOBOX IDC_LOGON_DOMAIN, 60, 95, 155, 14, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Con&firmă", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
-    PUSHBUTTON "A&nulează", IDCANCEL, 160, 122, 50, 14
-    PUSHBUTTON "&Oprește", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
+    PUSHBUTTON "&OK", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
+    PUSHBUTTON "&Revocare", IDCANCEL, 160, 122, 50, 14
+    PUSHBUTTON "&Oprire", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:48]: PUSHBUTTON "&OK", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
has accelerator collision with
PUSHBUTTON "&Oprire", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
Check you Romanian Windows!

File: dll/win32/msgina/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: I fixed it. Look in the last commit.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 3 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:167]: I fixed the &OK and the O&K issues. Now it's the same everywhere, please recheck. Unfortunately, I cannot send you any screenshot, as I cannot find msgina.dll in my computer.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 4 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:48]: Please, recheck the last commit.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 3 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:167]: I checked both %systemroot%\system32 and %systemroot%\SysWOW64 and I got nothing related to this file.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: I do confirm that The collisions are gone now. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 3 above)
Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc:167]: I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 4 above)
Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc:48]: I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.

File: dll/win32/msgina/lang/ro-RO.rc [hunk 5]
```diff
@@ -58,12 +58,12 @@ BEGIN
...
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
+    PUSHBUTTON "&Deautentificare…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
+    PUSHBUTTON "Închid&ere…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
+    PUSHBUTTON "Schimba&re parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
+    PUSHBUTTON "&Manager de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
+    PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:66]: Screenshot of Romanian Windows is required! You are obviously only guessing here!
  PUSHBUTTON "Schimba&re parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
collides with
 PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14

**Comment by AndreiMiloiu** [comment]
I don't know where to find Romanian 2k3.

**Comment by JoachimHenze** [comment]
> I don't know where to find Romanian 2k3. I tried to find it, but no result.

Then download a Romanian Windows XP instead, it uses to > 95 % the same sources as 2k3, and is easier to find in localized manner. let me google it for you...
You may be upgrade your Windows XP with the following package in case yours is not localized to romanian by default:
https://archive.org/details/WinXPLanguagePacks

**Comment by AndreiMiloiu** [comment]
> > I don't know where to find Romanian 2k3. I tried to find it, but no result.
> 
> Then download a Romanian Windows XP instead, it uses to > 95 % the same sources as 2k3, and is easier to find in localized manner. let me google it for you... You may be upgrade your Windows XP with the following package in case yours is not localized to romanian by default: https://archive.org/details/WinXPLanguagePacks If you do not have a localized Windows at hand, then please refrain from ever changing any accelerators ever! Because that would make the translation worse than it was before. And it is very tedious to review such changes line by line and in multiple iterations.

Thank you! I will look on it later.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:66]: I wasn't able to find exactly the place where are the words OK and Revocare, but I found they don't have an accelerator. I can send you a screenshot with what I found. These two words don't have any accelerator, so I will remove it everywhere.

Windows XP Professional-2024-02-26-01-20-25

Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: Yes, that looks like the best choice then to remove the accels. I do agree.

**Review by JoachimHenze — APPROVED** [review]
Approved now.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: There is a mistake in the Romanian translation that I didn't notice until now. These two words have no accelerator in any file, so every Romanian file must be modified. As you told me, I will try to do less PRs in the future, but with higher quality. It's now that I discover the mistakes.

Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: That's a good attitude. The problem with accelerator collisions is, that it does not only render 1 accelerators useless (the one touched), but also it always breaks a 2nd one (the one which it collides).
This conversation is resolved. I just left it uncollapsed because it shows the screenshot.

Comment (JoachimHenze) [reply dll/win32/msgina/lang/ro-RO.rc]: One last question: In your screenshot we can see that &Numere has an accelerator in the source code, but the preview doesn't show it. Maybe you have them configured to not always be shown in your Windows XP!
Since you did not find the exact OK and Revocare buttons (since they might come from the parent control...)

I do recommend you to configure your Windows XP to always show them:
In XP, right-click on the desktop, hit "Properties," head to the Appearance tab, click the "Effects" button, then un-check the "Hide underlined letters ..." option. Now your toolbar menus will always have their keyboard access letters underlined.

Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: I'm ready. Please, retest. Please, tell me where is wrong with the word &Numere. I heavily modified rasdlg.dll file in order to look better to XP, so I removed the unnecessary accelerator. Please let me know where is wrong. Maybe it was fixed these days.

Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: > One last question: In your screenshot we can see that &Numere has an accelerator in the source code, but the preview doesn't show it. Maybe you have them configured to not always be shown in your Windows XP! Since you did not find the exact OK and Revocare buttons (since they might come from the parent control...)
> 
> I do recommend you to configure your Windows XP to always show them: In XP, right-click on the desktop, hit "Properties," head to the Appearance tab, click the "Effects" button, then un-check the "Hide underlined letters ..." option. Now your toolbar menus will always have their keyboard access letters underlined.

I modified msgina and rasdlg files in order to be both as the XP version ones. Please retest to see if the problem persists.

File: dll/win32/msgina/lang/ro-RO.rc [hunk 6]
```diff
@@ -16,7 +16,7 @@ FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     CONTROL "", IDC_BAR, "Static", SS_OWNERDRAW, 0, 44, 275, 4
-    LTEXT "", IDC_STATUS_MESSAGE, 7, 59, 261, 12, SS_WORDELLIPSIS
+    LTEXT "", IDC_STATUS_Mesaj, 7, 59, 261, 12, SS_WORDELLIPSIS
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:19]: This is definitely a bug! Please revert that!

File: dll/win32/msgina/lang/ro-RO.rc [hunk 7]
```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
...
-    LTEXT "<Message>", IDC_SECURITY_MESSAGE, 15, 62, 245, 18
-    LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
-    CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentificare…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&ere…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schimba&re parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Manager de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
+    GROUPBOX "Informaţii de Log on", IDC_STATIC, 7, 50, 261, 52
+    LTEXT "<Mesaj>", IDC_SECURITY_Mesaj, 15, 62, 245, 18
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:58]: Bug!Bug! Do not rename the IDC_

File: dll/win32/msgina/lang/ro-RO.rc [hunk 8]
```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
...
+CAPTION "Computer blocat"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     ICON IDI_LOCKICON, IDC_STATIC, 7, 59, 20, 20
-    LTEXT "Acest calculator este în uz și a fost blocat.", IDC_STATIC, 36, 61, 232, 8
-    LTEXT "<Message>", IDC_LOCKED_MESSAGE, 37, 75, 231, 26
-    LTEXT "Apăsați simultan «Ctrl»+«Alt»+«Del» pentru a debloca acest calculator.", IDC_STATIC, 36, 106, 232, 8
+    LTEXT "Acest computer este activ şi a fost blocat.", IDC_STATIC, 36, 61, 232, 8
+    LTEXT "<Mesaj>", IDC_LOCKED_Mesaj, 37, 75, 231, 26
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:77]: Bug! Do not rename the IDC_

File: dll/win32/msgina/lang/ro-RO.rc [hunk 9]
```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
...
-CAPTION "Deblocare calculator"
+CAPTION "Deblocare computer"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     ICON IDI_LOCKICON, IDC_STATIC, 7, 59, 20, 20
-    LTEXT "Acest calculator este în uz și a fost blocat.", IDC_STATIC, 36, 61, 232, 8
-    LTEXT "<Message>", IDC_UNLOCK_MESSAGE, 36, 75, 232, 26
+    LTEXT "Acest computer este activ şi a fost blocat.", IDC_STATIC, 36, 61, 232, 8
+    LTEXT "<Mesaj>", IDC_UNLOCK_Mesaj, 36, 75, 232, 26
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:89]: Bug! IDC_ renamed!

File: dll/win32/msgina/lang/ro-RO.rc [hunk 10]
```diff
@@ -220,30 +220,30 @@ STRINGTABLE
 BEGIN
     IDS_LOGGEDOUTSAS "Bun venit!"
     IDS_LOCKEDSAS "Momentan calculatorul este blocat."
-    IDS_PRESSCTRLALTDELETE "Pentru autentificare, apăsați «Ctrl»+«Alt»+«Del»."
+    IDS_PRESSCTRLALTDELETE "Pentru Log on, apăsați Ctrl-Alt-Delete."
     IDS_ASKFORUSER "Nume utilizator: "
     IDS_ASKFORPASSWORD "Parola: "
     IDS_FORCELOGOFF "Această acțiune va deautentifica utilizatorului curent cu posibile pierderi ale datelor nesalvate. Continuați?"
-    IDS_LOCKMSG "Doar %s sau un Administrator poate debloca acest calculator."
+    IDS_LOCKMSG "Numai %s\\%s (%s) sau un administrator au posibilitatea să deblocheze acest computer."
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:227]: Post a screenshot of that line is en-US! Unless it has 3 %s as well, this might be a bug.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: dll/win32/msgina/lang/ro-RO.rc (hunk 6 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: I reverted it. I didn't pay attention.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: This is for msgina. I send you what you asked me before.



2024-03-11 15_44_39-Dialog - 1500

File: dll/win32/msgina/lang/ro-RO.rc (hunk 10 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:227]: This is for en-US

Windows XP Professional-2024-03-11-15-48-07

File: dll/win32/msgina/lang/ro-RO.rc [hunk 11]
```diff
@@ -16,7 +16,7 @@ FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     CONTROL "", IDC_BAR, "Static", SS_OWNERDRAW, 0, 44, 275, 4
-    LTEXT "", IDC_STATUS_MESSAGE, 7, 59, 261, 12, SS_WORDELLIPSIS
+    LTEXT "", IDC_LOCKED_MESSAGE, 7, 59, 261, 12, SS_WORDELLIPSIS
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:19]: Why did you change IDC_STATUS_MESSAGE to IDC_LOCKED_MESSAGE here?

File: dll/win32/msgina/lang/ro-RO.rc [hunk 12]
```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
...
-CAPTION "Deblocare calculator"
+CAPTION "Deblocare computer"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     ICON IDI_LOCKICON, IDC_STATIC, 7, 59, 20, 20
-    LTEXT "Acest calculator este în uz și a fost blocat.", IDC_STATIC, 36, 61, 232, 8
-    LTEXT "<Message>", IDC_UNLOCK_MESSAGE, 36, 75, 232, 26
+    LTEXT "Acest computer este activ şi a fost blocat.", IDC_STATIC, 36, 61, 232, 8
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 36, 75, 232, 26
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:89]: Why did you change IDC_UNLOCK_MESSAGE to IDC_LOCKED_MESSAGE here?

File: dll/win32/msgina/lang/ro-RO.rc (hunk 11 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: Sorry, I didn't pay attention. I mistakenly did it.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 12 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: Sorry, I didn't pay attention. I mistakenly did it.

File: dll/win32/msgina/lang/ro-RO.rc [hunk 13]
```diff
@@ -54,76 +54,76 @@ CAPTION "Securitate"
...
-    LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
-    CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    GROUPBOX "Informaţii de Log on", IDC_STATIC, 7, 50, 261, 52
+    LTEXT "<Mesaj>", IDC_LOCKED_MESSAGE, 15, 62, 245, 18
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:58]: you changed IDC_SECURITY_MESSAGE to IDC_LOCKED_MESSAGE here.

File: dll/win32/msgina/lang/ro-RO.rc [hunk 14]
```diff
@@ -220,47 +220,47 @@ STRINGTABLE
...
     IDS_SHUTDOWN_HIBERNATE "Hibernare"
     /* Shut down descriptions */
-    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori autentificarea în sistem."
-    IDS_SHUTDOWN_SHUTDOWN_DESC "Închide atât sesiunea curentă cât și sistemul de operare pentru a puta opri în siguranță alimentarea electrică."
-    IDS_SHUTDOWN_RESTART_DESC "Închide sesiunea curentă și repornește sistemul."
-    IDS_SHUTDOWN_SLEEP_DESC "Induce sistemul în starea de veghe."
-    IDS_SHUTDOWN_HIBERNATE_DESC "Salvează sesiunea curentă și oprește calculatorul."
+    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori Log on în sistem."
+    IDS_SHUTDOWN_SHUTDOWN_DESC "Se încheie sesiunea şi se închide Windows astfel înc\xE2t oprirea alimentării cu curent să se facă în siguranţă."
+    IDS_SHUTDOWN_RESTART_DESC "Se închide  sesiunea, se închide Windows, apoi se porneşte din nou ReactOS."
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:263]:
```suggestion
    IDS_SHUTDOWN_RESTART_DESC "Se închide sesiunea, se închide Windows, apoi se porneşte din nou ReactOS."
```

File: dll/win32/msgina/lang/ro-RO.rc [hunk 15]
```diff
@@ -220,47 +220,47 @@ STRINGTABLE
...
     IDS_SHUTDOWN_SLEEP "Stare de veghe"
     IDS_SHUTDOWN_HIBERNATE "Hibernare"
     /* Shut down descriptions */
-    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori autentificarea în sistem."
-    IDS_SHUTDOWN_SHUTDOWN_DESC "Închide atât sesiunea curentă cât și sistemul de operare pentru a puta opri în siguranță alimentarea electrică."
-    IDS_SHUTDOWN_RESTART_DESC "Închide sesiunea curentă și repornește sistemul."
-    IDS_SHUTDOWN_SLEEP_DESC "Induce sistemul în starea de veghe."
-    IDS_SHUTDOWN_HIBERNATE_DESC "Salvează sesiunea curentă și oprește calculatorul."
+    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori Log on în sistem."
+    IDS_SHUTDOWN_SHUTDOWN_DESC "Se încheie sesiunea şi se închide Windows astfel înc\xE2t oprirea alimentării cu curent să se facă în siguranţă."
```
Comment (JoachimHenze) [inline dll/win32/msgina/lang/ro-RO.rc:262]: I personally thing the old string was superior here, because it didn't need the \x in înc\xE2t. Is that really a word? But that is just an opinion. I am no native speaker.

Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:262]: You're right. I forgot to change it. I will correct it later.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 13 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:58]: I will change it back. I didn't pay attention.

File: dll/win32/msgina/lang/ro-RO.rc (hunk 15 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: Fixed!

File: dll/win32/msgina/lang/ro-RO.rc (hunk 13 above)
Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: Fixed!

File: dll/win32/msgina/lang/ro-RO.rc [hunk 16]
```diff
@@ -187,11 +187,11 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL IDI_DLG_BG, IDC_DLG_BG, "STATIC", SS_BITMAP | SS_REALSIZECONTROL, 0, 0, 208, 122
     CONTROL IDI_ROSLOGO_FLAG, IDC_ROSLOGO_WHITE, "STATIC", SS_BITMAP, 176, 1, 32, 26
-    PUSHBUTTON "&Hibernare", IDC_BUTTON_HIBERNATE, 30093, 30049, 22, 20, BS_OWNERDRAW | WS_DISABLED
+    PUSHBUTTON "&Hibernate", IDC_BUTTON_HIBERNATE, 30093, 30049, 22, 20, BS_OWNERDRAW | WS_DISABLED
```
Comment (GeoB99) [inline dll/win32/msgina/lang/ro-RO.rc:190]: Why un-translate it to English?

Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc:190]: Sorry, I mistakenly did this. I will revert this change.

Comment (AndreiMiloiu) [reply dll/win32/msgina/lang/ro-RO.rc]: It's fixed now!
